index.html

```diff
@@ -166,5 +216,33 @@ <h3>LET'S CONNECT</h3>
                 <br>
             </ul>
 </section>
-</body>
-</html>
\ No newline at end of file
+<footer>
+    <form action="https://formspree.io/f/xrgdkyvk" method="post">
```

Comment:
## I kindly suggest that you sticks to the design templates in Figma as much as possible, this includes the font, colours, images and spacing (margin/padding) by:
- [ ] Please ensure your skills and contact section do not overlap because in the figma design there is a big space between them.
- [ ] Please add the border radius to the top left side of the contact form as seen in the Figma design.

| Your Design  | Figma Design |
| ------------- | ------------- |
| <img width="220" alt="image" src="https://user-images.githubusercontent.com/52934128/193851889-2f602be0-bb49-4fcf-92ab-6a05e510d0ce.png"> | <img width="244" alt="image" src="https://user-images.githubusercontent.com/52934128/193852100-69b03c7a-cf17-4dea-8554-2ea598769b43.png"> |